build/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM --platform=linux/amd64 golang:1.18
+FROM --platform=linux/amd64 buildpack-deps:bullseye-scm
+# we are using buidlpack-deps:bullseye-scm https://github.com/docker-library/golang/blob/8d0fa6028120904e16fe761f095bd0620b68eab2/1.18/bullseye/Dockerfile
 
-WORKDIR /opt/kubefirst-install
+ARG KUBEFIRST_VERSION=1.8.5
```

Comment:
same 1.8.6 comment here - i may be mistaken with the approach, or i may also just early reviewing the pr and this is set up so you can test these changes

Reply:
updated, thanks.